package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "astro-blog",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

This is an incomplete suggestion, but the scripts would be added here:

```suggestion
    "test": "echo \"Error: no test specified\" && exit 1",
    "lint": "",
    "spellcheck": ""
```